tests/common/fixtures/advanced_reboot.py

```diff
@@ -158,13 +164,19 @@ def getTestbedType(self):
         Accessor method for testbed's topology name
         """
         return self.tbinfo['topo']['name']
+    
+    def get_mux_status(self, duthost):
+        cmd = 'show mux status --json'
+        return json.loads(duthost.shell(cmd)['stdout'])
 
     def __buildTestbedData(self, tbinfo):
         """
         Build testbed data that are needed by ptf advanced-reboot.ReloadTest class
         """
 
         self.mgFacts = self.duthost.get_extended_minigraph_facts(tbinfo)
+        if DUAL_TOR_MODE:
+            self.other_mgFacts = self.other_duthost.get_extended_minigraph_facts(tbinfo)
```

Comment:
minor: can we rename `other_mgFacts` to `peer_mgFacts`?

Reply:
Done